WooCommerce/Classes/ViewRelated/Products/AI/ProductDescriptionGenerationView.swift

```diff
@@ -167,8 +199,14 @@ private extension ProductDescriptionGenerationView {
         )
         static let insertGeneratedText = NSLocalizedString("Apply",
                                                            comment: "Button title to insert AI-generated product description.")
-        static let generateText = NSLocalizedString("Generate",
+        static let generateText = NSLocalizedString("Write with AI",
                                                     comment: "Button title to generate product description with Jetpack AI.")
+        static let regenerateText = NSLocalizedString("Regenerate",
+                                                      comment: "Button title to regenerate product description with Jetpack AI.")
+        static let sampleFeatures = NSLocalizedString(
+            "Example: Potted, Cactus, Plant, Decorative, Easy-care",
```

Comment:
super nit: would it be more natural when the words in the example are only capitalized for the first word since that's the capitalization style if they type the same words to the text field?

Reply:
I suppose design wanted to use uppercase to make the words stand out. But it makes sense to keep them lowercased to be more realistic. Updated in d51ac58590d6786d7e1bb94e64771c90ca1fcff3.